src/framework/analyses.ml

```diff
@@ -119,6 +118,100 @@ struct
     | x -> BatPrintf.fprintf f "<analysis name=\"fromspec\">%a</analysis>" printXml x
 end
 
+(* Tuple of fundec and S.C*)
+module T (Base1: Printable.S) (Base2: Printable.S) =
```

Comment:
This seems to be mostly a `Printable.Prod`? We should be able to get some reuse there... Also `T` is a very generic name if you then have `fundec` and `context`

Reply:
We actually searched for a module that implements tuples, but we haven't found it... 
I now deleted module T and reused the Printable.Prod

Reply:
Done